general_prep.md

```diff
@@ -8,6 +8,8 @@ Pretend you're about to come down with a debilitating yet temporary disease.
 4. Stock up on easy to make foods. Soups, crackers, pasta, ready-to-bake pizza, frozen veggies.
 5. Get your errands done before you go on-call. Buy pet food, get groceries, pick up the dry cleaning, etc. You may not have time to do it during your rotation, and you don't want to lug all your equipment around while you do it. It is okay to take off thirty minutes early to get these errands done before you go on-call.
 
+Be cognizant of your SLO/RTO and plan any activities accordingly.  That being said, do not abuse the SLO/RTO.  If you have a 15 minute window to respond/acknowledge your page, do so as soon as reasonably possible.  Never intentionally wait until the latest allowed moment.
```

Comment:
I really like this section, but some people might not know what an SLO or RTO is. Can you link to an explanation?